.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,14 +4,20 @@ about: Suggest an idea for this project
 
 ---
 
-## Description
+## Problem
 
-A short summary of the idea.
+What is the user, security, system, or project sustainability need this Issue was created to address? Is there evidence to cite in feedback from users, contributors, or instance owners? Is there evidence to cite in a report from a domain expert, in ux or dev best practices, or in user research findings? Or, is this something you simply feel compelled to address?
 
-## User Research Evidence
+## Beneficiaries
 
-Does evidence exist from user research to support this idea? Could research or design help validate or improve it? Has this idea been influenced by your conversations with users, or by your own experience as a SecureDrop user? If so, please note it here, and [take a look at our UX wiki](https://github.com/freedomofpress/securedrop-ux/wiki) for more information about our design and research efforts.
+Who will directly benefit from the implementation of a solution? Users (Sources, Journalists, Admins, or Installers?), Instance owners, project contributors (FPF or volunteers), the project’s steward (FPF), or another group? Ref [SecureDrop Users](https://github.com/freedomofpress/securedrop-ux/wiki#SecureDrop_Users) if needed.
```

Comment:
This asks for user stories without being explicit about it. They're a widely-understood concept by developers these days, so the section below would cover it. Also again it feels a bit gate-keepery, especially by narrowly defining the system's users for the prospective requestor and referring them to further reading if they don't get it. Also, there's that steward word again, of which I am personally not a fan.

Reply:
Wayl... I'm intentionally trying to bring developers out of developer-world things, and into thinking about users from a purely human-centric perspective. I don't feel that Agile does the best job of this, either, by reducing human-centricity to "Billy wants to fetch a pail of water." WHY does Billy want to do that, as an example, is ignored per Agile and dev standards. 

"Gatekeepring" is totally not the goal, and I appreciate you throwing that flag. I would like to keep contributors focused on who we'd like to optimize the product for, however—but not to the extreme of outright exclusion. Thoughts?

What would you propose as an alternative word to "Steward" to reference FPF with—as FPF doesn't explicitly "own" the product/code?